airflow/api_connexion/openapi/v1.yaml

```diff
@@ -4943,6 +4956,11 @@ components:
           items:
             type: string
           description: The value can be repeated to retrieve multiple matching values (OR condition).
+        executor:
```

Comment:
you do allow multiple executors to be filtered at once, right? If so, should it names "executors"?

Reply:
The others like pool and queue go by this nomenclature as well, so I think we should leave in singular. And I suppose they're also trying to match the name of the ti field precisely.